rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -99,16 +102,19 @@
               tempest = load 'pipeline_steps/tempest.groovy'
               holland = load 'pipeline_steps/holland.groovy'
               maas = load 'pipeline_steps/maas.groovy'
+              kibana = load 'pipeline_steps/kibana.groovy'
           }}
           dir("rpc-gating/playbooks") {{
             writeFile file: "inventory/hosts", text: "[job_nodes]\n${{INSTANCE_NAME}} ansible_host=${{INSTANCE_IP}}\n"
             common.install_ansible()
           }}
           ssh_slave.connect()
           node(INSTANCE_NAME) {{
-            sh """
-               lvcreate -n aio -L250G lxc
-            """
+            if (env.STAGES.contains("Prepare LEM AIO")) {{
```

Comment:
Actually this is a good question and has me now thinking whether what I'm doing is right.  Give me a sec to think about this a bit more!

Side note, `Prepare Multi-Node AIO` does end up appearing as a stage on https://rpc.jenkins.cit.rackspace.net/job/LEM-Multi-Node-AIO_master-xenial-periodic/.